flake.nix

```diff
@@ -49,36 +45,67 @@
             pname = "jank";
             version = "git";
 
-            # Add only essential files so that the source hash is consistent.
```

Comment:
Lots of formatting and other small changes in the PR. You should run `nix fmt flake.nix` and avoid changing unrelated lines.